README.md

```diff
@@ -118,6 +118,80 @@ Example for a subject like `Foo::Bar#baz` it will run all example groups with de
 `Foo::Bar#baz`, `Foo::Bar` and `Foo`. The order is important, so if mutant finds example groups in the
 current prefix level, these example groups *must* kill the mutation.
 
+Reading Mutation Output
```

Comment:
Lets name this "Reading Mutant Output", because the output is specific to mutants mutation testing implementation and not generic.